modules/ROOT/pages/oauth2-policies-new.adoc

```diff
@@ -3,62 +3,27 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-// this repeats some info from external-oauth-2.0-token-validation-policy, but we need an opener page
+To secure an API in Anypoint Platform using OAuth 2.0, you can use the following policies and the matching OAuth 2.0 provider:
```

Comment:
DITTO previous GLOBAL about standardizing the presentation of this name:

OAuth 2.0 Provider
the OAuth 2.0 provider
Mule OAuth 2.0 Provider
the Mule OAuth 2.0 provider

Reply:
Done! Thanks!